bin/ruby-build

```diff
@@ -1020,18 +1020,7 @@ has_broken_mac_openssl() {
   is_mac || return 1
   local openssl_version="$(/usr/bin/openssl version 2>/dev/null || true)"
   [[ $openssl_version = "OpenSSL 0.9.8"?* || $openssl_version = "LibreSSL"* ]] &&
-  [[ "$RUBY_CONFIGURE_OPTS" != *--with-openssl-dir=* ]] &&
-  ! use_homebrew_openssl
-}
-
-use_homebrew_openssl() {
```

Comment:
Can you leave this function as-is? Custom definitions may be referencing it

Reply:
good point - had not thought of that!